### DISCUSSION TITLE
Time drifting on EasyTouch2

### DISCUSSION THREAD

**Discussion by npawelek** [tagyoureit/nodejs-poolController#946] [Q&A] [answered]
Is there a recommended way to manage the time through njspc for my controller? My RPi is using NTP to remain current at the OS level, but no matter what I choose, manual or server, my time is off in the dashpanel by a couple of minutes. I have an easytouch2 panel and I know there is a clock adjust functionality for drift, but I haven't tracked the drift to know how much it shifts over time.

I'm curious how other folks are doing it and what is the preferred way to set and manage this properly.

**Comment by tagyoureit** [comment, marked as answer]
The code is here: https://github.com/tagyoureit/nodejs-poolController/blob/master/controller/comms/messages/status/EquipmentStateMessage.ts#L195

njsPC will adjust the time when the drift is >5 minutes.

**Comment by npawelek** [comment]
Ah interesting—I'll let it do it's thing and see how it goes. Appreciate it!
